FlowCrypt/src/main/java/com/flowcrypt/email/model/Screenshot.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.model
+
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * @author Denis Bondarenko
+ *         Date: 8/8/22
+ *         Time: 2:23 PM
+ *         E-mail: DenBond7@gmail.com
+ */
+data class Screenshot(val byteArray: ByteArray) : Parcelable {
+  constructor(parcel: Parcel) : this(parcel.createByteArray() ?: byteArrayOf())
+
+  override fun equals(other: Any?): Boolean {
+    if (this === other) return true
+    if (javaClass != other?.javaClass) return false
+
+    other as Screenshot
+
+    if (!byteArray.contentEquals(other.byteArray)) return false
+
+    return true
```

Comment:
last 3 non-empty lines of code can be just `return byteArray.contentEquals((other as Screenshot).byteArray)`

Reply:
Agree. But this code was generated by Android Studio. I would like to leave it as-is. Later this class can be changed and modification will be simpler if we leave it as-is.